match/lib/match/storage/google_cloud_storage.rb

```diff
@@ -0,0 +1,245 @@
+require 'fastlane_core/command_executor'
+require 'fastlane_core/configuration/configuration'
+require 'google/cloud/storage'
+
+require_relative '../options'
+require_relative '../module'
+require_relative './interface'
+
+module Match
+  module Storage
+    # Store the code signing identities in on Google Cloud Storage
+    class GoogleCloudStorage < Interface
+      DEFAULT_KEYS_FILE_NAME = "gc_keys.json"
+
+      # User provided values
+      attr_accessor :type
+      attr_accessor :platform
+      attr_accessor :bucket_name
+      attr_accessor :google_cloud_keys_file
+
+      # Managed values
+      attr_accessor :gc_storage
+
+      # Append Google Cloud specific options to `Match::Options`
+      Match::Options.append_option(
```

Comment:
What is the reason for appending these here? 🤔 Might be difficult for another contributor to find these as other options are all put into `options.rb`. I think these would _probably_ fit better in `options.rb` but I would probably be okay with a comment in `options.rb`